Formula/rtx.rb

```diff
@@ -24,6 +24,14 @@ def install
     generate_completions_from_executable(bin/"rtx", "completion")
   end
 
+  def caveats
+    <<~EOS
+      To activate rtx, add this to your shell profile (e.g.: ~/.bashrc or ~/.zshrc):
+          eval "$(rtx activate [SHELL])"
+      Replace [SHELL] with your shell (bash, zsh, fish, etc.)
+    EOS
+  end
```

Comment:
This is not Homebrew-specific documentation. We no long add information that belongs in upstream documentation to `caveats`.